.travis.yml

```diff
@@ -30,17 +30,15 @@ jobs:
       name: "Bazel: //src_kt/... "
     - script: tools/run_in_docker "bazel --bazelrc=tools/travis.bazelrc build //src/... && bazel --bazelrc=tools/travis.bazelrc test //src/..."
       name: "Bazel: //src/..."
-    - script: tools/run_in_docker "bazel --bazelrc=tools/travis.bazelrc build //particles/... && bazel --bazelrc=tools/travis.bazelrc test //particles/..."
-      name: "Bazel: //particles/..."
+    - script: tools/run_in_docker "bazel --bazelrc=tools/travis.bazelrc build //particles/... --build_tag_filters kotlinc && bazel --bazelrc=tools/travis.bazelrc test //particles/... --build_tag_filters kotlinc  && tools/sigh testShells"
+      name: "Bazel: //particles/... (Kotlin) + testShells"
+    - script: tools/run_in_docker "bazel  --bazelrc=tools/travis.bazelrc build //particles/... --build_tag_filters emscripten && bazel --bazelrc=tools/travis.bazelrc test //particles/... --build_tag_filters emscripten && tools/sigh testShells"
+      name: "Bazel: //particles/... (C++) + testShells"
     # run ktlint against every kotlin file in the diff between the branch and master's head.
     - script: tools/run_in_docker "git diff --name-only --relative $travis_commit_range | grep '\.kt[s\"]\?$' | xargs ktlint --android --relative --verbose ."
       name: "KtLint"
-    - script: tools/run_in_docker "tools/sigh test"
-      name: "Node Test: Unit"
-    - script: tools/run_in_docker "tools/sigh testShells"
-      name: "Node Test: Shells"
-    # - script: tools/run_in_docker "npm run build:rollup && npm --prefix=server test"
-    #   name: "Node Test: Server"
+    - script: tools/run_in_docker "tools/sigh lint && tools/sigh test"
+      name: "Sigh lint & test"
```

Comment:
Just checking: still not running `tslint` ... is `tslint` deprecated or something?

Reply:
This is running tools/sigh lint. The last I checked, this runs two lint subcommands, including tslint.